tests/test_fields.py

```
@@ -0,0 +1,138 @@
+from pytest import raises
```

Ruff (0.12.2)

Incorrect import of `pytest`; use `import pytest` instead

(PT013)